### DISCUSSION TITLE
如何发送临时消息

### DISCUSSION THREAD

**Discussion by FrankOu2001** [Mrs4s/go-cqhttp#1584] [Q&A] [answered]
我用nonebot2编写了一个QQ机器人，适配器是onebotv11，但是其自带的`send_private_msg`函数无法向没有添加好友的成员发送临时消息。go-cqhttp的`send_private_msg`似乎能实现发送临时消息，那么请问我该怎样在python中调用？

**Comment by Pandaft** [comment, marked as answer]
#1338

**Reply by FrankOu2001** [reply]
所以是没有办法让机器人通过临时会话发送消息了？

**Reply by Pandaft** [reply]
go-cqhttp 生成的配置文件默认不开启该功能，参考配置文件第 17 行。
https://docs.go-cqhttp.org/guide/config.html#%E9%85%8D%E7%BD%AE%E4%BF%A1%E6%81%AF

**Reply by FrankOu2001** [reply]
感谢，已经解决了。我通过临时会话向机器人发送指令，结果失败了。后来加了好友之后就能用了
